Packs/CortexResponseAndRemediation/Playbooks/silent-Playbook-Azure_AD_Administrator_Role_Assignment.yml

```diff
@@ -0,0 +1,2813 @@
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "31"
+      - "33"
+      - "15"
+      - "9"
+      USER:
+      - "57"
+    separatecontext: false
+    conditions:
+    - label: USER
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: Core.OriginalAlert.event.identity_type
+            iscontext: true
+          right:
+            value:
+              simple: USER
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1495,
+          "y": 846
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "15":
+    id: "15"
+    taskid: 6f5f1d34-2049-4762-87c7-e94ea01f0d8d
+    type: title
+    task:
+      id: 6f5f1d34-2049-4762-87c7-e94ea01f0d8d
+      version: -1
+      name: Azure Risk Detections
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "28"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1272.5,
+          "y": 1123
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "22":
+    id: "22"
+    taskid: b8859ac0-f081-4912-ad4c-95cccc1ab3ad
+    type: regular
+    task:
+      id: b8859ac0-f081-4912-ad4c-95cccc1ab3ad
+      version: -1
+      name: Get Azure Risk Detections
+      description: Returns a list of the **Risk Detection** objects and their properties from the **past day**
+      script: '|||azure-risky-users-risk-detections-list'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "24"
+    scriptarguments:
+      detected_date_time_after:
+        simple: ${.=new Date(Date.now()-(1000*60*60*24)).toISOString()}
+      limit:
+        simple: "100"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1023,
+          "y": 1391
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "24":
+    id: "24"
+    taskid: 2b46833e-c0b1-427e-aa50-a0a182819a01
+    type: regular
+    task:
+      id: 2b46833e-c0b1-427e-aa50-a0a182819a01
+      version: -1
+      name: Save Assignee-Triggered Detections
+      description: where the **risk state** is either *atRisk* or *confirmedCompromised* and the **risk level** is *medium* or *high*
```

Comment:
Where the **risk state** is either *atRisk* or *confirmedCompromised* and the **risk level** is *medium* or *high*